WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsV4PeriodViewController.swift

```diff
@@ -156,6 +156,11 @@ final class StoreStatsV4PeriodViewController: UIViewController {
         observeReloadChartAnimated()
     }
 
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        restartGhostAnimationIfNeeded()
```

Comment:
Minor nit: Maybe we should add a comment for this method to explain why it's necessary here in the `viewWillAppear` method?

Reply:
Good idea! Added in https://github.com/woocommerce/woocommerce-ios/pull/5823/commits/65166f8c84b9aff5fcbd5d9294c4320990f8178a